website/content/docs/glossary.mdx

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+page_title: Glossary of Terms
+sidebar_title: Glossary
+description: |-
+  Vault Glossary.
+---
+
+# Glossary
+
+This page collects brief definitions of some of the technical terms used in the
+documentation for Vault.
+
+- [Audit Device](#audit-device)
+- [Auth Method](#auth-method)
+- [Barrier](#barrier)
+- [Client Token](#client-token)
+- [Secret](#secret)
+- [Secrets Engine](#secrets-engine)
+- [Server](#server)
+- [Storage Backend](#storage-backend)
+
+### Audit Device
+
+An audit device is responsible for managing audit logs.
+Every request to Vault and response from Vault goes through the configured
+audit devices. This provides a simple way to integrate Vault with multiple
+audit logging destinations of different types.
+
+### Auth Method
+
+An auth method is used to authenticate users or applications
+which are connecting to Vault. Once authenticated, the auth method returns the
+list of applicable policies which should be applied. Vault takes an
+authenticated user and returns a client token that can be used for future
+requests. As an example, the `userpass` auth method uses a username and
+password to authenticate the user. Alternatively, the `github` auth method
+allows users to authenticate via GitHub.
+
+### Barrier
+
+The barrier is cryptographic steel and concrete around the
+Vault. All data that flows between Vault and the storage backend passes
+through the barrier. The barrier ensures that only encrypted data is written
+out, and that data is verified and decrypted on the way in. Much like a bank
+vault, the barrier must be "unsealed" before anything inside can be accessed.
+
+### Client Token
+
+A client token (aka "Vault Token") is conceptually
+similar to a session cookie on a web site. Once a user authenticates, Vault
+returns a client token which is used for future requests. The token is used by
+Vault to verify the identity of the client and to enforce the applicable ACL
+policies. This token is passed via HTTP headers.
+
+### Secret
+
+A secret is the term for anything returned by Vault which
+contains confidential or cryptographic material. Not everything returned by
+Vault is a secret, for example system configuration, status information, or
+policies are not considered secrets. Secrets always have an associated lease.
+This means clients cannot assume that the secret contents can be used
+indefinitely. Vault will revoke a secret at the end of the lease, and an
+operator may intervene to revoke the secret before the lease is over. This
+contract between Vault and its clients is critical, as it allows for changes
+in keys and policies without manual intervention.
+
+### Secrets Engine
+
+A secrets engine is responsible for managing secrets.
+Simple secrets engines like the "kv" secrets engine simply return the same
```

Comment:
```suggestion
Simple secrets engines, such as the "kv" secrets engine, return the same
```
Per the style guide, I believe we avoid "simply", "just", and other oversimplifying words.